Mage/src/main/java/mage/game/permanent/token/ThopterColorlessToken2.java

```diff
@@ -0,0 +1,29 @@
+package mage.game.permanent.token;
+
+import mage.MageInt;
+import mage.abilities.keyword.FlyingAbility;
+import mage.constants.CardType;
+import mage.constants.SubType;
+
+public class ThopterColorlessToken2 extends TokenImpl {
```

Comment:
I prefer something more descriptive than tacking on a number, maybe "Thopter00ColorlessToken" ? Not critical